app/vmctl/README.md

```diff
@@ -11,9 +11,12 @@ Features:
 - [x] OpenTSDB: migrate data from OpenTSDB to VictoriaMetrics
 - [ ] Storage Management: data re-balancing between nodes
 
-vmctl acts as a proxy between data source ([Prometheus](#migrating-data-from-prometheus),
+vmctl provides various useful actions with VictoriaMetrics components. 
```

Comment:
Would be better to move this info (that vmctl provides migration and other options) to the top of the readme file

Reply:
Changed top of the readme file